pom.xml

```diff
@@ -302,7 +214,7 @@
 
     <properties>
         <identity.framework.version>5.15.0</identity.framework.version>
-        <identity.sso.agent.package.export.version>${project.version}</identity.sso.agent.package.export.version>
+        <asgario.tomcat.saml.agent.version>${project.version}</asgario.tomcat.saml.agent.version>
```

Comment:
Whats the usage of this property?

Reply:
We are using in the sample app pom.xml to add tomcat agent as a dependency of the webapp

Reply:
Can't we use project version there directly?

Reply:
Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/2/commits/8c6cf400b66a4e63ec7226c85a6884b7e379fae1